src/window.rs

```diff
@@ -218,20 +221,18 @@ impl WindowWrapper {
 
     pub fn handle_pointer_motion(&mut self, x: i32, y: i32) {
         let previous_position = self.mouse_position;
-        if let Ok(new_mouse_position) = LogicalSize::from_physical_size_tuple(
-            (
-                (x as f32 / self.renderer.font_width) as u32,
-                (y as f32 / self.renderer.font_height) as u32,
-            ),
-            &self.window,
-        ) {
-            self.mouse_position = new_mouse_position;
-            if self.mouse_down && previous_position != self.mouse_position {
-                BRIDGE.queue_command(UiCommand::Drag(
-                    self.mouse_position.width,
-                    self.mouse_position.height,
-                ));
-            }
+        let physical_size = PhysicalSize::new(
```

Comment:
Why not just do LogicalSize::new here? Did that not compile?

Reply:
I just assumed that the ` pub fn handle_pointer_motion(&mut self, x: i32, y: i32) {` receives x,y as PhysicalSize and that needs converting to LogicalSize. Therefore, I'm doing PhysicalSize::new and then converting with PhysicalSize.to_logical(1.0) .

Now, I agree with you. I am not sure this is the best way (and the 1.0 working suggest that LogicalSize and PhysicalSize are the same). But I didn't try LogicalSize to be honest.

Reply:
I personally can't test this, because I just realized now that I don't even have the mouse functionality for moving the cursor. As long as it works and is more readable, it should be good to commit.

Reply:
Just tested on OSX, works.

Reply:
The scaling factor is about the dpi scaling. This will work if you don't have any scaling enabled, but if the screen you are on has some sort of scaling, you'll run into issues. I believe we will have to create a skulpin window wrapper here and ask it for the scale factor

Reply:
https://github.com/aclysma/skulpin/blob/master/skulpin-renderer-sdl2/src/lib.rs#L33

Reply:
Yeah. You're right. We should ask the system what the scaling is and do that. 

On Linux I have "scaling" by xrandr into the desired resolution. And i think OSX does it 'transparent' to applications. 

We might have to test this a bit better, let me know if you need help on this.

Reply:
I can do the fix after this merges. But yes I believe this will need manual testing on all the platforms. I suspect there will be necessary upstream changes as well